zerver/views/video_calls.py

```diff
@@ -188,7 +188,10 @@ def deauthorize_zoom_user(request: HttpRequest) -> HttpResponse:
 
 @has_request_variables
 def get_bigbluebutton_url(
-    request: HttpRequest, user_profile: UserProfile, meeting_name: str = REQ()
+    request: HttpRequest,
+    user_profile: UserProfile,
+    meeting_name: str = REQ(),
+    is_audio_only: bool = REQ(json_validator=check_bool, default=False),
```

Comment:
Let's call this `voice_only` -- we renamed them to "voice calls" in the UI a few weeks ago.